src/components/MapPopup/MapPopup.styles.tsx

```diff
@@ -64,3 +64,15 @@ export const Color = styled.div<{ color: string; $percent: number }>`
   border: solid thin #cdcdcd;
   margin-left: 0;
 `;
+
+export const List = styled.ul`
+  width: 95%;
```

Comment:
I personally don't  like when we give such specific values to width - like 95%. I know that this was to solve this issue, right?

![image](https://github.com/user-attachments/assets/4979328a-067b-4c02-8530-343d9b4224ac)

But to solve that we could add a 

```suggestion
   padding-right: 0.5rem;
   box-sizing: border-box;
```
It's a better way to solve this, it ultimately makes kind of the same thing but it's easier to debug in the future.